pkgs/build-support/setup-hooks/make-wrapper-auto/default.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, dieHook
+, makeSetupHook
+, jq
+}:
+
+{
+  __functor = makeSetupHook {
+    deps = [ dieHook ];
+    substitutions = {
+      jq = "${lib.getBin jq}/bin/jq";
+    };
+  } ./hook.sh;
+
+  combineWrappersInfo = { envInfo
+    , symlink ? {}
+    , buildInputs ? []
+    , propagatedBuildInputs ? []
+  }: let
+    inputsWrapInfo = lib.concatStringsSep " " (
+      builtins.map (input: "${lib.getDev input}/nix-support/wrappers.json") (
+        buildInputs ++ propagatedBuildInputs
+      )
+    );
+    envInfoFile = builtins.toFile "env-info.json" (builtins.toJSON envInfo);
+  in /* sh */ ''
```

```suggestion
  in /* bash */ ''
```

otherwise shellcheck will complain about [[ being undefined.